platform/broadcom/sonic-platform-modules-cel/blackstone/modules/cls-fpga.c

```diff
@@ -20,30 +20,31 @@
 #include <linux/interrupt.h>
```

Comment:
Please run the checkpatch.pl on this file and address the complaints.  
I saw the codes style is updated but some of them are against the Linux kernel code style.